sockeye/checkpoint_decoder.py

```diff
@@ -140,7 +141,7 @@ def decode_and_evaluate(self,
             for i, input_sentence in enumerate(self.input_sentences):
                 tic = time.time()
                 trans_input = translator.make_input(i, input_sentence)
-                trans_output = translator.translate(trans_input)
+                trans_output = translator.translate([trans_input])[0]
```

Comment:
mhh, now that the batch size is 16, we should also feed in sentence in chunks here, no? Why not just feed in the entire input_sentences?

Reply:
Agreed.
On a related note, we could also cache the results of make_input but that could be done in another PR.

Reply:
This will break the percentile calculation, no? I think this was why I left batch size 1 initially for checkpoints. As a workaround, we could report some surrogate percentiles, approximated from batch translation times and batch_size.